Source/WebCore/dom/Document.cpp

```diff
@@ -8907,7 +9067,24 @@ void Document::mediaStreamCaptureStateChanged()
     });
 }
 
-#endif
+void Document::updateVideoCaptureAccordingMicrophoneInterruption(bool isMicrophoneInterrupted)
```

Comment:
`...According...` isn't quite correct. Maybe either `...AccordingTo...` or `updateVideoCaptureStateForMicrophoneInterruptionState`?

Reply:
Will change to `updateVideoCaptureStateForMicrophoneInterruption`